### ISSUE TITLE
Improve alt text for product images

### ISSUE THREAD

**Issue by NthabiNonjo** [NthabiNonjo/Task1#1]
The alt text for product images should be specific to each product for better accessibility.

**Comment by NthabiNonjo** [comment]
Done with issue
